ProcessLib/HT/HTFEM.h

```diff
@@ -306,6 +306,8 @@ class LocalAssemblerData : public HTLocalAssemblerInterface
             auto const& N = ip_data.N;
             auto const& dNdx = ip_data.dNdx;
 
+            pos.setIntegrationPoint(ip);
```

Comment:
This can be directly be pushed to the master. Has nothing to do with Richards...